openhtf/core/test_record.py

```diff
@@ -70,7 +70,7 @@ def add_outcome_details(self, code, description=''):
 
 class PhaseRecord(  # pylint: disable=no-init
     mutablerecords.Record(
-        'PhaseRecord', ['descriptor_id', 'name', 'codeinfo'],
+        'PhaseRecord', ['descriptor_id', 'name', 'codeinfo', 'options'],
```

Comment:
If you make this optional, we stay compatible with previous serialized `TestRecord` objects